assets/js/base/components/loading-mask/index.js

```diff
@@ -4,12 +4,12 @@
 import { __ } from '@wordpress/i18n';
 import PropTypes from 'prop-types';
 import classNames from 'classnames';
-import { Spinner } from 'wordpress-components';
 
 /**
  * Internal dependencies
  */
 import './style.scss';
+import Spinner from '../spinner';
```

Comment:
We had two types of spinner across blocks so I swapped to our version with a different loading icon.